subliminal/providers/legendastv.py

```diff
@@ -469,7 +474,7 @@ def query(self, language, title, season=None, episode=None, year=None):
                 # iterate over releases
                 for r in releases:
                     subtitle = self.subtitle_class(language, t['type'], t['title'], t.get('year'), t.get('imdb_id'),
-                                                   t.get('season'), a, r)
+                                                   t.get('season'), a, r, a.featured)
```

Comment:
If it is named featured here it should be named featured in the Subtitle class too.